lib/verify.js

```diff
@@ -54,17 +54,25 @@ export default async (pluginConfig, context) => {
   }
 
   if (gitlabToken && projectPath) {
+    let tokenHeader;
     let projectAccess;
     let groupAccess;
 
     logger.log("Verify GitLab authentication (%s)", gitlabApiUrl);
 
+    switch(true) {
+      case str.startsWith("glcbt-"):
+        tokenHeader = "JOB-TOKEN"
+      default:
+        tokenHeader = "PRIVATE-TOKEN"
+    }
```

Comment:
```suggestion
    switch(gitlabToken) {
      case gitlabToken.startsWith("glcbt-"):
        tokenHeader = "JOB-TOKEN"
      default:
        tokenHeader = "PRIVATE-TOKEN"
    }
```

@AlexandreEXFO - you need to switch the gitlabToken here.